test/attribute.js

```diff
@@ -140,6 +140,7 @@ test('`element` attributes', async (t) => {
       }
     )
 
+    // TODO: why? "hidden" is  a valid value for the `hidden` attribute
```

Comment:
Because `hidden` looks better? Because `hidden="hidden"` is longer and superfluous? Because it’s how people often write HTML?

This package is also used in combination of minifiers / pritifyers. So it’s not solely about what “works”.

🤔 I’d say this comment can be removed?